dev/automated_tests/flutter_test/README.md

```diff
@@ -1,5 +1,5 @@
-The files in this directory are used as part of tests in the
+The files in this directory are used as part of the tests in the
 `flutter_tools` package. Some are here because here these tests need a
 `pubspec.yaml` that references the flutter framework (which is
 intentionally not true of the `flutter_tools` package). Others are
-here mostly out of peer pressure.
+here mostly out of peer pressure.
```

Comment:
what's the diff here?

Reply:
Try highlighting both together. The extra space after the period is removed.

Reply:
LGTM. again, weird the diffing algorithm didn't highlight (it usually highlights trailing whitespace changes).

Reply:
I think because they are doing updates to the site currently, so maybe sometimes things don't work correctly